+brille/private/required_modules.m

```diff
@@ -0,0 +1,3 @@
+function out = required_modules()
+    out = {'brille', 'TO_BE_DETERMINED'};
```

Comment:
Will leaving `'TO_BE_DETERMINED'` here throw an error in `brille_on` or `install_python_modules`?

Reply:
Ah... I should have noted that `TO_BE_DETERMINED` and `TO_BE_REPLACED` are both substituted in `release.m` for the actual package. If you're running with a "development" version (e.g. in the main repo directory) the `TO_BE_DETERMINED` will cause an error. I'll just rename it `0.0.0`.

Reply:
`'TO_BE_DETERMINED'` only appears in `required_modules.m` if you run, e.g,

``` sh
find . -type f -exec grep -H "TO_BE" {} \;
```

you get the output:
```
./+brille/private/required_modules.m:    out = {'brille', 'TO_BE_DETERMINED'};
./mltbx/brillem.prj:    <param.version>TO_BE_REPLACED</param.version>
./release.m:    text = regexprep(text, 'TO_BE_REPLACED', version);
```

And I see now that you replace the whole contents of `required_modules.m` when running `release`, so I guess indicating only the major version, `0`, that the (unreleased) package is intended to work with might be sufficient since `verify_python_modules` *should* expand `0` to the equivalent of `0.0.0`.

Reply:
yeah, it was such a short file it was easier just to rewrite it... but I'll change it so it does a proper string replacement in case we want to change the contents of `required_modules.m` later.